src/index.js

```diff
@@ -1,19 +1,19 @@
 // @flow
-'use strict';
```

Comment:
ESLint assumes we're in ES6 module, that's why it removes it. Because we're inside a Node module, not ES6, we need to add `'use strict';`.
This rule will make it work: 
```
strict: ['error', 'global']
```